tests/specs/functional/lemmas-spec.k

```diff
@@ -602,6 +602,17 @@ module LEMMAS-SPEC
         => doneLemma ( X ) ... </k>
         requires #rangeUInt ( 256 , X )
 
+ // #padRightToWidth
+ // ----------------
+
+    claim [pRTW-01]:
+      <k> runLemma (
+            notBool ( #asWord ( #padRightToWidth ( 32 , #buf ( 4 , VV0_x_114b9705:Int ) ) ) ==Int
+              #asWord ( #buf ( 4 , VV0_x_114b9705:Int ) +Bytes b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00" ) )
+          ) => doneLemma(
+            false
+          ) ... </k>
```

Comment:
I guess there should be a side-condition on `VV0_x_...` from the original unsimplified term:

```suggestion
    claim [pRTW-01]:
      <k> runLemma (
            notBool ( #asWord ( #padRightToWidth ( 32 , #buf ( 4 , VV0_x_114b9705:Int ) ) ) ==Int
              #asWord ( #buf ( 4 , VV0_x_114b9705:Int ) +Bytes b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00" ) )
          ) => doneLemma(
            false
          ) ... </k>
        requires #rangeUInt(32, VV0_x_114b9705)
```